Починить шлем гладиатора

MorgoWizard opened the pull request:
Удалили возможность поднимать забрало, т.к. не хватает иконок для всех рас

MorgoWizard committed:
Починить шлем гладиатора

Удалили возможность поднимать забрало, т.к. не хватает иконок для всех рас

Bizzonium commented on code/modules/clothing/head/helmet.dm:
```diff
@@ -185,12 +185,6 @@
 	flags = BLOCKHAIR
 	item_state = "gladiator"
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES
-	toggle_message = "You attach the face shield to the"
```
Замени удаление строк на комментирование и описанием почему кнопку отключили

MorgoWizard committed:
Обновить иконки шлемов для рас

Использовал нарисованные иконки, обновил список рас, которые должны использовать иконки и вернул кнопку смены забрала шлема.

MorgoWizard commented:
пока незаконченно

XaycBC commented:
#62 
мне кажется они нашли друг друга

MorgoWizard commented:
Не актуально
